tools/docker/ci/Dockerfile.tf_serving

```diff
@@ -0,0 +1,41 @@
+ARG OCCLUM_VERSION
+
+FROM tensorflow/serving:latest-devel as binary_build
```

Comment:
Reuse the existing Dockerfile is better than copy it.

Reply:
Done